migrator/migrations/m_187_to_m_188_remove_reports_without_migrated_collections/schema/convert_collections.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by pg-bindings generator. DO NOT EDIT.
+package schema
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// ConvertResourceCollectionFromProto converts a `*storage.ResourceCollection` to Gorm model
+func ConvertResourceCollectionFromProto(obj *storage.ResourceCollection) (*Collections, error) {
+	serialized, err := obj.Marshal()
+	if err != nil {
+		return nil, err
+	}
+	model := &Collections{
+		ID:            obj.GetId(),
+		Name:          obj.GetName(),
+		CreatedByName: obj.GetCreatedBy().GetName(),
+		UpdatedByName: obj.GetUpdatedBy().GetName(),
+		Serialized:    serialized,
+	}
+	return model, nil
+}
+
+// ConvertResourceCollection_EmbeddedResourceCollectionFromProto converts a `*storage.ResourceCollection_EmbeddedResourceCollection` to Gorm model
+func ConvertResourceCollection_EmbeddedResourceCollectionFromProto(obj *storage.ResourceCollection_EmbeddedResourceCollection, idx int, collectionID string) (*CollectionsEmbeddedCollections, error) {
+	model := &CollectionsEmbeddedCollections{
+		CollectionsID: collectionID,
+		Idx:           idx,
+		ID:            obj.GetId(),
+	}
+	return model, nil
+}
```

Comment:
unused; remove?

Reply:
This is an autogenerated file. I'd rather leave it as is since some functions are in use